api/cluster/templater.go

```diff
@@ -275,6 +275,7 @@ func (t *KFServingResourceTemplater) createTransformerSpec(modelService *models.
 	envVars = append(envVars, models.EnvVar{Name: envTransformerPort, Value: defaultTransformerPort})
 	envVars = append(envVars, models.EnvVar{Name: envTransformerModelName, Value: modelService.Name})
 	envVars = append(envVars, models.EnvVar{Name: envTransformerPredictURL, Value: createPredictURL(modelService)})
+	envVars = append(envVars, models.EnvVar{Name: envPredictorStorageURI, Value: utils.CreateModelLocation(modelService.ArtifactURI)})
```

Comment:
@tiopramayudi This should be enough to mount model artifact in transformer right?

Reply:
transformer container name also must be changed to `kfserving-container`

Reply:
This will break logging isn't it? @tiopramayudi